### PR TITLE
Add documentation to standalone agent.

### PR DESCRIPTION
**Pull request by schandrika**
Updated documentation of examples/StandAloneListener agent
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/VOLTTRON/volttron/pull/1841%23discussion_r234325487%22%2C%20%22https%3A//github.com/VOLTTRON/volttron/pull/1841%23discussion_r234326521%22%2C%20%22https%3A//github.com/VOLTTRON/volttron/pull/1841%23discussion_r234327585%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%20adced64fd53e1492d621a9e44cdb1721e5be0daa%20examples/StandAloneListener/standalonelistener.py%205%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/VOLTTRON/volttron/pull/1841%23discussion_r234325487%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22There%20should%20be%20a%20preamble%20before%20starting%20at%201.%22%2C%20%22created_at%22%3A%20%222018-11-16T19%3A40%3A16Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/3979063%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/craig8%22%7D%7D%2C%20%7B%22body%22%3A%20%22fixed%22%2C%20%22created_at%22%3A%20%222018-11-16T19%3A47%3A44Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/16124920%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/schandrika%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20examples/StandAloneListener/standalonelistener.py%3AL1-39%22%7D%2C%20%22Pull%20adced64fd53e1492d621a9e44cdb1721e5be0daa%20examples/StandAloneListener/standalonelistener.py%2036%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/VOLTTRON/volttron/pull/1841%23discussion_r234326521%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22sorry%20fixed%20it.%22%2C%20%22created_at%22%3A%20%222018-11-16T19%3A43%3A55Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/16124920%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/schandrika%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20examples/StandAloneListener/standalonelistener.py%3AL1-39%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-Pull adced64fd53e1492d621a9e44cdb1721e5be0daa examples/StandAloneListener/standalonelistener.py 5'></a> &nbsp;<b><a href='https://github.com/VOLTTRON/volttron/pull/1841#discussion_r234325487'>File: examples/StandAloneListener/standalonelistener.py:L1-39</a></b>
- <a href='https://github.com/craig8'></a> There should be a preamble before starting at 1.
- <a href='https://github.com/schandrika'></a> fixed
- [x] <a href='#crh-comment-Pull adced64fd53e1492d621a9e44cdb1721e5be0daa examples/StandAloneListener/standalonelistener.py 36'></a> &nbsp;<b><a href='https://github.com/VOLTTRON/volttron/pull/1841#discussion_r234326521'>File: examples/StandAloneListener/standalonelistener.py:L1-39</a></b>
- <a href='https://github.com/schandrika'></a> sorry fixed it.

<a href='https://www.codereviewhub.com/VOLTTRON/volttron/pull/1841?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/VOLTTRON/volttron/pull/1841?approve=1'></a>&nbsp;<a href='https://github.com/VOLTTRON/volttron/pull/1841'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by craig8** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/VOLTTRON/volttron/pull/1841'>Refresh</a>.<a href='#crh-update-none'></a>

File: examples/StandAloneListener/standalonelistener.py [hunk 1]
```diff
@@ -1,11 +1,38 @@
...
+    roles (delimit multiple entries with comma) []:
+    groups (delimit multiple entries with comma) []:
+    mechanism [CURVE]:
+    credentials []: GsEq7mIsU6mJ31TN44lQJeGwkJlb6_zbWgRxVo2gUUU
+    comments []:
+    enabled [True]:
+
+    4. With a volttron activated shell this script can be run like:
+
+       python test_weather2.py
```
Comment (craig8) [inline examples/StandAloneListener/standalonelistener.py]: Shouldn't this use standalonelistener.py not your weather agent?

File: examples/StandAloneListener/standalonelistener.py [hunk 2]
```diff
@@ -1,11 +1,38 @@
 '''
-        This python script will listen to the defined vip address for specific
-    topics.  The user can modify the settings.py file to set the specific
-    topics to listen to.
```
Comment (craig8) [inline examples/StandAloneListener/standalonelistener.py]: There should be a preamble before starting at 1.

File: examples/StandAloneListener/standalonelistener.py (hunk 1 above)
Comment (schandrika) [reply examples/StandAloneListener/standalonelistener.py]: sorry fixed it.

File: examples/StandAloneListener/standalonelistener.py (hunk 2 above)
Comment (schandrika) [reply examples/StandAloneListener/standalonelistener.py]: fixed

**Pull request merged**